CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.7.1
+
+Release date: August 30, 2017
+
+This release contains a security update and it is highly recommended that users upgrade to this version.
```

Comment:
True, but [our disclosure policy](https://www.mattermost.org/responsible-disclosure-policy/) states that we shouldn't disclose details of a vulnerablity until 14 days after the release, and this could be considered as such.